pkg/network/dhcpv4/BUILD.bazel

```diff
@@ -7,7 +7,7 @@ go_library(
         "ethtool.go",
         "socket_listener.go",
     ],
-    importpath = "kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/network/dhcp",
+    importpath = "kubevirt.io/kubevirt/pkg/network/dhcpv4",
```

Comment:
Can you please explain the motivation of this change?
dhcp (4 and 6) are pure virt-launcher code.
I see the new `pkg\network` as a place to locate code that is not clear if it belongs to virt-launcher or virt-handler. Like common methods in use by `phase1` and `phase2`.